### PR TITLE
pkg(com.google.android.setupwizard): improve description

### PR DESCRIPTION
**Pull request by AnonymousWP**

### CODE REVIEW

**Review by Rudxain — DISMISSED** [review]

**Comment by Rudxain** [comment]
It may be worth mentioning that the breakage only happens if there wasn't a Google account in "Accounts". That's why I could still sync, because I uninstalled SW after adding my account

**Comment by AnonymousWP** [comment]
Better now?

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -29255,7 +29255,7 @@
   },
   "com.google.android.setupwizard": {
     "list": "Google",
-    "description": "Android Setup\nThe factory reset device basic configuration setup guides you through the basics of setting up your device.\nWARNING: Oddly enough, disabling/uninstalling this package will break mobile identity management which could be used by apps (for example your bank) to authenticate you. Disabling/uninstalling may also break (Google) account synchronisation. See https://en.wikipedia.org/wiki/Mobile_identity_management for Mobile Identity Management.",
+    "description": "Android Setup\nThe factory reset device basic configuration setup guides you through the basics of setting up your device.\nWARNING: Oddly enough, disabling/uninstalling this package will break mobile identity management which could be used by apps (for example your bank) to authenticate you. Disabling/uninstalling may also break account syncing (if the Google account is absent). See https://en.wikipedia.org/wiki/Mobile_identity_management for Mobile Identity Management.",
```
Comment (Rudxain) [inline resources/assets/uad_lists.json:29258]: The wording is somewhat confusing 😕.  Something along the lines of "breaks syncing of newly-added Google accounts" or "breaks syncing of Google accounts setup after this pack is removed" would be better, IMO

**Review by Rudxain — CHANGES_REQUESTED** [review]

File: resources/assets/uad_lists.json (hunk 1 above)
Comment (AnonymousWP) [reply resources/assets/uad_lists.json:29258]: Should be better now.

**Review by Rudxain — APPROVED** [review]

**Pull request merged**